zigpy_cc/types.py

```diff
@@ -73,7 +74,7 @@ class ParameterType(uint8_t, enum.Enum):
 class NetworkOptions:
     def __init__(self) -> None:
         self.networkKeyDistribute = False
-        self.networkKey = [1, 3, 5, 7, 9, 11, 13, 15, 0, 2, 4, 6, 8, 10, 12, 13]
-        self.panID = 0x1A62
-        self.extendedPanID = [0xDD, 0xDD, 0xDD, 0xDD, 0xDD, 0xDD, 0xDD, 0xDD]
-        self.channelList = [11]
+        self.networkKey = [0x6A, 0x8F, 0xBB, 0x75, 0xB2, 0xE8, 0xD9, 0x6D, 0xE5, 0xC5, 0x2D, 0x9D, 0x18, 0x56, 0x41, 0x1D]
+        self.panID = 0x8258
+        self.extendedPanID = [0x7A, 0xB2, 0x1E, 0x59, 0x19, 0x55, 0x46, 0x27]
+        self.channelList = [Common.LogicalChannel.CH15]
```

Comment:
Why did you change network options?

Reply:
Even after a complete flash erase and re-flash, I could not form a network. Changing the PAN ID helped.

It would be better to have a way to configure these values directly from the controller application so that the codebase isn't polluted with these defaults, which I'll probably try doing in a bit.

Reply:
A better question I think is why all of these settings are hard-coded to begin with. The bellows project picks a random pan ID and extended pan ID when forming a new network. I believe the security key is picked at random as well.